### PR TITLE
Gerrit integration [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by m-kostrzewa**
**DO NOT MERGE**

Original author is Jarek. I just rebased.

### CODE REVIEW

File: CIScripts/InitializeCIScript.ps1 [hunk 1]
```diff
@@ -5,6 +5,8 @@ if ($Env:ENABLE_TRACE -eq $true) {
 
 # Refresh Path
 $Env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine")
+$Env:PSModulePath
```
Comment (wurbanski) [inline CIScripts/InitializeCIScript.ps1]: Potrzebujemy tego wypisywania? brzydko wygląda

File: CIScripts/InitializeCIScript.ps1 [hunk 2]
```diff
@@ -5,6 +5,8 @@ if ($Env:ENABLE_TRACE -eq $true) {
...
+$Env:Path
```
Comment (wurbanski) [inline CIScripts/InitializeCIScript.ps1]: Potrzebujemy tego wypisywania? brzydko wygląda

File: CIScripts/InitializeCIScript.ps1 (hunk 1 above)
Comment (wurbanski) [inline CIScripts/InitializeCIScript.ps1]: Potrzebujemy tego wypisywania? brzydko wygląda

File: CIScripts/InitializeCIScript.ps1 (hunk 2 above)
Comment (m-kostrzewa) [reply CIScripts/InitializeCIScript.ps1]: feel free to add commits to this PR, let's treat it as a working branch

**Review by wurbanski — DISMISSED** [review]
Ok

**Comment by m-kostrzewa** [comment]
Janie, przetestuj proszę

File: CIScripts/BuildAll.ps1 [hunk 3]
```diff
@@ -2,9 +2,49 @@
 . $PSScriptRoot\BuildFunctions.ps1
 . $PSScriptRoot\Job.ps1
 
+$ToolsAllowBranchOverride = $true
+
+# Additional logic for builds triggered from Gerrit
+if (Test-Path Env:GERRIT_CHANGE_ID)  {
```
Comment (m-kostrzewa) [inline CIScripts/BuildAll.ps1:8]: IMO if anything, we should specify repo name in param (like `contrail-sandesh`), and then build the URL (prepend) with github/gerrit and then CodiLime or Juniper

**Comment by m-kostrzewa** [comment]
@wurbanski @climebil https://github.com/codilime/contrail-windows-tools/pull/110 <- let's use this branch, as I rebased it to refactor branch there

**Comment by krdln** [comment]
Since we've moved to #110 I don't see a reason to keep it open.

**Pull request closed without merging**